bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -21,14 +21,38 @@ function ListPage() {
   });
 
   //  this.state = {establisments:{}}
-
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  console.log(urlParams.get('zones'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7143775)
```suggestion
  console.log(urlParams.get("zones"));
```